lib/new_relic/harvest/telemetry_sdk/dimensional_metrics/harvester.ex

```diff
@@ -77,14 +77,12 @@ defmodule NewRelic.Harvest.TelemetrySdk.DimensionalMetrics.Harvester do
          %{type: :count, name: name, value: new_value, attributes: attributes} = metric,
          metrics_acc
        ) do
-    attributes_hash = :erlang.phash2(attributes)
```

Comment:
This is an interesting one. My concern was the memory used to store the map of attributes in full has part of the key, thus using a hashing of the attributes.

I realize that `phash2` can have collisions, but I figured it was unlikely given the name and attributes that would be used within a harvest cycle of 10 seconds (I think that's the default).

Reply:
Spoke internally with everyone and we can just ignore me!